src/ageAssurance/components/NoAccessScreen.tsx

```diff
@@ -0,0 +1,340 @@
+    _(
+      msg`You are accessing Bluesky from a region that legally requires us to verify your age before allowing you to access the app.`,
+    ),
+  ]
+
+  const onPressLogout = useCallback(() => {
+    if (isWeb) {
+      // We're switching accounts, which remounts the entire app.
+      // On mobile, this gets us Home, but on the web we also need reset the URL.
+      // We can't change the URL via a navigate() call because the navigator
+      // itself is about to unmount, and it calls pushState() too late.
+      // So we change the URL ourselves. The navigator will pick it up on remount.
+      history.pushState(null, '', '/')
+    }
+    logoutCurrentAccount('AgeAssuranceNoAccessScreen')
+  }, [logoutCurrentAccount])
+
+  const birthdayUpdateText = canUpdateBirthday ? (
+    <Text style={[textStyles]}>
+      <Trans>
+        If your birth date is not accurate, you can update it by{' '}
+        <SimpleInlineLinkText
+          label={_(msg`Click here to update your birthday`)}
+          style={[textStyles]}
+          {...createStaticClick(() => {
+            birthdayControl.open()
+          })}>
+          clicking here
+        </SimpleInlineLinkText>
+        .
+      </Trans>
+    </Text>
+  ) : null
+
+  return (
+    <>
+      <ScrollView
+        contentContainerStyle={[
+          a.px_2xl,
+          {
+            paddingTop: isWeb ? a.p_5xl.padding : insets.top + a.p_2xl.padding,
+            paddingBottom: 100,
+          },
+        ]}>
+        <View
+          style={[
+            a.mx_auto,
+            a.w_full,
+            web({
+              maxWidth: 380,
+              paddingTop: gtPhone ? '8vh' : undefined,
+            }),
+            {
+              gap: 32,
+            },
+          ]}>
+          <View style={[a.align_start]}>
+            <AgeAssuranceBadge />
+          </View>
+
+          {hasDeclaredAge ? (
+            <>
+              {isAARegion ? (
+                <>
+                  <View style={[a.gap_lg]}>
+                    {blocks.map((block, index) => (
+                      <Text key={index} style={[textStyles]}>
+                        {block}
+                      </Text>
+                    ))}
+
+                    {birthdayUpdateText}
+                  </View>
+
+                  <AccessSection />
+                </>
+              ) : (
+                <View style={[a.gap_lg]}>
+                  <Text style={[textStyles]}>
+                    <Trans>
+                      Unfortunately, the birth date you have saved to your
+                      profile makes you too young to access Bluesky.
+                    </Trans>
+                  </Text>
+
+                  {birthdayUpdateText}
+                </View>
+              )}
+            </>
+          ) : (
+            <View style={[a.gap_lg]}>
+              <Text style={[textStyles]}>
+                <Trans>
+                  Looks like you haven't added your birth date. You must provide
```

Comment:
```suggestion
                  It looks like you haven't added your birthdate. You must provide
```
nit: as above and suggest slight rewording